typings/numba/types/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+class Type: ...
+class Number(Type): ...
+
+class Integer(Number):
+    def __init__(self, name: str) -> None: ...
+
+class CPointer(Type):
+    def __init__(self, dtype: Type) -> None: ...
+
+uint32 = Integer("uint32")
+uint64 = Integer("uint64")
+void = none
```

Comment:
None?

Reply:
Oh I guess this is `none = NoneType("none")` but I am surprised it doesn't need an import from `core.types`

Reply:
I think I can just do `None`